test/Microsoft.AspNet.Mvc.DataAnnotations.Test/DataAnnotationsModelValidatorTest.cs

```diff
@@ -259,23 +356,23 @@ public void Validate_IsValidFalse_StringLocalizerReturnsLocalizerErrorMessage()
             // Arrange
             var metadata = _metadataProvider.GetMetadataForType(typeof(string));
             var container = "Hello";
-            var model = container.Length;
 
-            var attribute = new Mock<ValidationAttribute> { CallBase = true };
-            attribute.Setup(a => a.IsValid(model)).Returns(false);
+            var attribute = new MaxLengthAttribute(4);
+            attribute.ErrorMessage = "Length";
```

Comment:
Use an error message that looks reasonable e.g. a slight variant of the default `[MaxLength]` message.